modules/configuration/src/test/java/org/apache/ignite/internal/configuration/ConfigurationRegistryTest.java

```diff
@@ -104,6 +106,23 @@ void testValidationPolymorphicConfigurationExtensions() {
         
         configRegistry.stop();
     }
+
+    @Test
+    void polymorphicSchemasShouldBeRequiredIfPolymorphicIdIsUsed() {
```

Comment:
Let's rename to `missingPolymorphicExtension`

Reply:
Your suggestion does not specify the condition that is important here: that there is a field annotated with @PolymorphicId. How about `missingPolymorphicExtensionWithPolymorhicIdIsPresent`?

Reply:
This is not a valid check, it should be if you have a polymorphic configuration, then there should be extensions for it.

Reply:
Renamed it to `polymorphicSchemasShouldBeRequiredIfPolymorphicExtensionsAreUsed`. Is it ok or do you still insist on `missingPolymorphicExtension`?